tests/src/UtilTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Test;
+
+use PHPUnit\Framework\TestCase;
+
+final class UtilTest extends TestCase
+{
+    public function testSeparateList()
+    {
+        $expected = ['x', 10, 20, 30, 40, 50];
+
+        $this->assertEquals($expected, separateList("	x,10;20 30|40	50,	"));
+        $this->assertEquals($expected, separateList('x, 10, 20, 30, 40, 50'));
+        $this->assertEquals($expected, separateList("x	10	20	30	40	50"));
```

Comment:
might be better to use escape sequences so the tabs don't accidentally get replaced by spaces by someone's editor

Reply:
Good point, changed 👍